pyproject.toml

```diff
@@ -90,13 +90,14 @@ test-min = [
     "pytest>=4.4",
     "pytest-nunit",
     "profimp",
+    "scanpy[dask]",
+    "scanpy[dask-ml]",
```

Comment:
please undo.

the min tests are specifically there to test scanpy with no optional dependencies.

if they don’t run, you need to do one or more of the following:.

- add a `@needs('...')` to tests that require some dependency
- move an import from module level to the inside of a scanpy function so scanpy can be imported without that dependency installed